src/guides/v2.3/extension-dev-guide/depend-inj.md

```diff
@@ -45,23 +45,34 @@ Service classes that do not exist in the codebase, such as [proxies], [factories
 The following code sample highlights the two types of dependency injections used in Magento:
 
 ```php
+
 namespace Magento\Backend\Model\Menu;
+
 class Builder
 {
+    /**
+     * @var Factory
+     */
+    private $menuItemFactory;
```

Comment:
This clarification makes sense. However, in the constructor we have the following assignment

```
$this->_itemFactory = $menuItemFactory;
```

So we need either declare `private _itemFactory` or rename `this->$_itemFactory`.

Also, please provide a fully qualified class name to the `Factory` in the DocBlock. Since we use fully qualified names everywhere in the class, it will be more understandable for new developers. 

Thank you!